src/parse.rs

```diff
@@ -9,7 +9,7 @@ use crate::alloc::vec::Vec;
 use crate::iter::HexToBytesIter;
 
 #[rustfmt::skip]                // Keep public re-exports separate.
-pub use crate::error::{HexToBytesError, HexToArrayError};
+pub use crate::error::{HexToBytesError, HexToArrayError, InvalidLengthError, OddLengthStringError, InvalidCharError};
```

Comment:
I'm not really sure we should export the innermost errors. It'll be rare that people will access them by name (they will call method on them instead).

Reply:
I had that exact same thought this morning when I woke up :)